install/app-fastfetch.sh

```diff
@@ -1,3 +1,5 @@
-sudo add-apt-repository -y ppa:zhangsongcui3371/fastfetch
+#!/usr/bin/env bash
+
+sudo add-apt-repository ppa:zhangsongcui3371/fastfetch
```

Comment:
shouldn't this line keep the `-y` flag to avoid prompting the user for confirmation?

Reply:
Right, I must have messed this up in my merge conflict. Thanks!